beta/tailwind.config.js

```diff
@@ -32,8 +32,16 @@ module.exports = {
       none: 'none',
     },
     extend: {
+      width:{
+        inherit : 'inherit'
+      },
+      maxHeight:{
+        illustration : '300px'
+      },
       maxWidth: {
-        xs: '21rem'
+        xs: '21rem',
+        inherit : 'inherit',
+        file : '160px'
```

Comment:
This feels like unnecessary one-off to me. I’d just keep this inline instead of introducing a project level changes.

i defer to @lebo to review this part.

Reply:
let's remove that because it looks like a built-in tailwind style. so somebody might expect other things to work, e.g. `m-inherit` but it won't